### ISSUE TITLE
MacOSX Monterey

### ISSUE THREAD

**Issue by raycoe** [sqlitebrowser/sqlitebrowser#3076]
Details for the issue
--------------------

#### What did you do?
Open DB4S and use SQL editor

#### What did you expect to see?
Standard formatting on query

#### What did you see instead?
A lot of problem in the editor that scrambling requests.

Useful extra information
-------------------------
> DB4S v3.12.2 [built for x86_64-little_endian-lp64] on macOS 10.16 (darwin/21.5.0) [x86_64]
> using SQLCipher Version 4.4.3 community (based on SQLite 3.34.1)
> and Qt 5.15.2

**Comment by chrisjlocke** [comment]
> A lot of problem in the editor that scrambling requests

We've not had other issues of this, but I'm not 100% clear on what the issue is from your description - are you saying the display is corrupted and you can't see results?
Can you provide a screenshot at all?
We have nightly builds for macOS which is a different version from the released 3.12.2 - can you try that one at all?
https://nightlies.sqlitebrowser.org/latest/

**Comment by raycoe** [comment]
I created a video screenshot that should better explain:
https://drive.google.com/file/d/1Yw-jTb92pQElgyEJEqS6O8BZMol9yqhr/view?usp=sharing

Problems are both in the editing and the results. In the video, columns
with "uuid" name are supposed to display UUIDv4 but something is wrong in
the display.

Hope this helps.
R

**Comment by chrisjlocke** [comment]
This looks like a font issue.

![image](https://user-images.githubusercontent.com/7872218/175549471-1148632d-a49c-43b4-a8a9-e59982d2ae15.png)

If you click 'Edit' and 'Preferences' then the 'Data browser' tab, change the font here to change the font for the grid.

If you click the 'SQL' tab, the font specified there changes the font in the SQL entry pane and the results pane.

**Comment by raycoe** [comment]
Indeed, by setting the fonts to courrier looks to be solving my issues.

Never had to change this, but ok.

Thx a lot
R

**Comment by chrisjlocke** [comment]
Some fonts have weird spacing and kerning, and Qt being cross-platform sometimes baulks at certain fonts at certain resolutions.  That font could work great on another system, so its really hit and miss.
Glad it worked for you though.
